network_monitor.py

```diff
@@ -59,7 +59,7 @@ def create_usage(ifs):
     for index, pcapy_device in enumerate(ifs):
         # if we are on windows, try and resolve the device UUID into an IP address.
         if sys.platform.startswith("win"):
-            from six.moves import winreg  # pytype: disable=import-error
+            import winreg  # pytype: disable=import-error
```

Comment:
pretty sure we can remove this warning ignore comment

Reply:
I think we still need that ignore comment. It prevents warnings being thrown on none Windows platforms where winreg is not available.

Reply:
Right, I'm fairly certain the comment must remain